pkg/gadgets/biolatency/gadget.go

```diff
@@ -100,8 +101,10 @@ func (t *Trace) Start(trace *gadgetv1alpha1.Trace) {
 	}
 
 	t.cmd = exec.Command("/usr/share/bcc/tools/biolatency")
-	t.out.Reset()
-	t.cmd.Stdout = &t.out
+	t.stdout.Reset()
+	t.stderr.Reset()
```

Comment:
Are these two calls really needed?

Reply:
Yes because the biolatency gadget can be stopped and restarted, so the same buffer can be reused for several commands.